recipes/CommonVoice/ASR/transformer/hparams/train_ar_hf_whisper.yaml

```diff
@@ -0,0 +1,151 @@
+# ################################
+# Model: Whisper (Encoder-Decoder) + NLL
+# Augmentation: TimeDomainSpecAugment
+# Authors: Pooneh Mousavi 2022
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/train_whisper/<seed>/<locale>
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the biggest Fairseq english whisper model.
+whisper_hub: openai/whisper-tiny
+test_only: False # Set it to True if you only want to  do the evaluation
+
+# Normalize inputs with
+# the same normalization done in the paper. Refer to Appendix C for further information.
```

Comment:
merge line 20 and 21 pls :-) + add the ref of the paper so that ppl knows the appendix you are talking about